packages/framework/build.xml

```diff
@@ -43,6 +43,7 @@
     <property name="translations.dump.flags" value="--keep"/>
     <property name="yaml-standards.indent-count" value="4"/>
     <property name="yaml-standards.args" value=""/>
+    <property name="path.stylelint.executable" value="node_modules/.bin/stylelint"/>
```

Comment:
Value should be `${path.root}/node_modules/.bin/stylelint` (same as eslint executable)
Please sort this property after `path.src` as they should be sorted alphabetically (one of our rules https://docs.shopsys.com/en/8.0/contributing/coding-standards/)

Reply:
Ok, fixed